### PR TITLE
Tweak: enlarge clickable area for overflow menu and dismiss watchlist tooltip when clicking on it

### PR DESCRIPTION
**Pull request by cooltey**
https://phabricator.wikimedia.org/T269448#6747169

### CODE REVIEW

File: app/src/main/java/org/wikipedia/page/PageActivity.java [hunk 1]
```diff
@@ -238,6 +241,9 @@ public void onShowTabsButtonClicked() {
 
     @OnClick(R.id.page_toolbar_button_show_overflow_menu)
     public void onShowOverflowMenuButtonClicked() {
+        if (watchlistTooltip != null) {
+            watchlistTooltip.dismiss();
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/page/PageActivity.java:245]: should we set it to `null` here also?

File: app/src/main/java/org/wikipedia/page/PageActivity.java [hunk 2]
```diff
@@ -238,6 +241,9 @@ public void onShowTabsButtonClicked() {
 
     @OnClick(R.id.page_toolbar_button_show_overflow_menu)
     public void onShowOverflowMenuButtonClicked() {
+        if (watchlistTooltip != null) {
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/page/PageActivity.java:244]: do we also want to check if it's `isShowing()`?

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/page/PageActivity.java (hunk 1 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/page/PageActivity.java:245]: Yes!

File: app/src/main/java/org/wikipedia/page/PageActivity.java (hunk 2 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/page/PageActivity.java]: Yes and done.

**Pull request merged**